docker-compose.yml

```diff
@@ -71,32 +72,25 @@ services:
     networks:
       - x11
 
-  foundry:
+  agd:
     profiles:
-      - foundry
-    container_name: foundry
-    image: synthetixio/foundry:457bb48776c3b14de232d9dda620ba9188dc40ac-base
-    command: [ 'anvil --no-cors --fork-url ${ANVIL_FORK_URL}' ]
-    environment:
-      - ANVIL_IP_ADDR=0.0.0.0
+      - synpress
+    container_name: agoric_chain
+    image: frazarshad/offerup-test-chain:1.0.0
+    platform: linux/amd64
```

Comment:
Just wanted to know that is `frazarshad/offerup-test-chain:1.0.0` intended to be a permanent image for our deployments, or can we expect to switch to another image provided by the company?

Reply:
no this a temporary solution until we have a better place to keep this image.